src/ui/Logo/Logo.ts

```diff
@@ -45,18 +67,14 @@ export class Logo extends Component {
       },
       SVGIcons.icons.coveoPoweredBy
     );
+    this.options.target ? link.setAttribute('target', this.options.target) : null;
```

Comment:
I wonder if the truthy check is needed? If `target` is set to `undefined`, empty string, does that change the behaviour?

If the check is needed, you could also use `&&` to remove the `else` part of the ternary while still keeping the logic concise.

Reply:
And ternaries outside an assignment statement look a bit off.